lab_2/main.py

```diff
@@ -43,7 +79,20 @@ def get_text_vector(original_text: list, language_profiles: dict) -> list or Non
     :param original_text: any tokenized text
     :param language_profiles: a dictionary of dictionaries - language profiles
     """
-    pass
+    if (not isinstance(original_text, list)) or (not isinstance(language_profiles, dict)):
+        return None
+    unique_words = get_language_features(language_profiles)
+    text_vector = []
+    for word in unique_words:
+        if word not in original_text:
+            text_vector.append(0)
+        elif word in original_text:
```

Comment:
The `word` is either `in` or `not` so I believe that the `else` block would be more appropriate here.